addons/account_edi_ubl_cii/models/account_move_send.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+import base64
+import logging
+import io
+
+from lxml import etree
+from xml.sax.saxutils import escape, quoteattr
+
+from odoo import _, api, fields, models, tools
+from odoo.tools import cleanup_xml_node
+from odoo.tools.pdf import OdooPdfFileReader, OdooPdfFileWriter
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountMoveSend(models.Model):
+    _inherit = 'account.move.send'
+
+    enable_ubl_cii_xml = fields.Boolean(compute='_compute_send_mail_extra_fields')
+    checkbox_ubl_cii_xml = fields.Boolean(
+        string="BIS Billing",
+        compute='_compute_checkbox_ubl_cii_xml',
+        store=True,
+        readonly=False,
+    )
```

Comment:
I think it would be better to have one checkbox per format, it's clearer for the user and it allows to generate multiple xml at once (for instance: ubl bis 3 and ubl A_NZ in Australia)

Reply:
we have 1 for cii and 1 for ubl